libs/language-server/src/lib/jayvee-module.ts

```diff
@@ -67,6 +68,10 @@ export const JayveeModule: Module<
       new JayveeHoverProvider(services),
   },
   RuntimeParameterProvider: () => new RuntimeParameterProvider(),
+  documentation: {
+    DocumentationProvider: (services: LangiumServices) =>
+      new JayveeDocumentationProvider(services),
```

Comment:
This custom provider might be worth reconsidering. Due to time pressure, I didn't check why the default one doesn't work. It should usually work with JSDoc. The `isJsDoc` function however returned `false` on the multi-line comments.

Reply:
Done! Now the following format is used for doc comments:

```
/**
* <my-doc>
*/